mk/config.mk

```diff
@@ -1159,6 +1159,21 @@ $(eval $(call cfg-depends-all,CFG_WIDEVINE_HUK,CFG_DT))
 CFG_WIDEVINE_PTA ?= n
 $(eval $(call cfg-depends-all,CFG_WIDEVINE_PTA,CFG_DT CFG_WIDEVINE_HUK))
 
+# Enable the semihosting console driver which inputs/outputs the trace messages
+# from/to the file or debug terminal on the semihosting host computer.
```

Comment:
Sorry nitpicking:  input console is supported only when using the semihosting debug console.
Proposal:
```
# CFG_SEMIHOSTING_CONSOLE, when enabled, embeds a semihosting console driver.
# When CFG_SEMIHOSTING_CONSOLE_FILE=NULL, OP-TEE console reads/writes
# characters from/to the debug terminal of the semihosting host computer.
# When CFG_SEMIHOSTING_CONSOLE_FILE="{your_log_file}", OP-TEE console
# outputs trace messages to that file. Output to "optee.log" by default.

Reply:
I changed `characters` to `trace messages`.
Please see https://github.com/OP-TEE/optee_os/pull/6700/commits/bd19a9ba6609366f28a70c8ec16a90cd5267ab81

Reply:
Hi @etienne-lms 
Would you like to review this again?